src/widget_builder.js

```diff
@@ -226,6 +227,26 @@ const buildAgGrid = (view, gridData, gridOptions_str, div, sheet, dropdownMulti
     if (view.model.get('sync_grid')) {
         exportFunc.exportGrid(gridOptions, view);
     }
+
+    // Listen to scroll property changes from python side
+    view.model.on('change:scroll', () => {
+        let scroll = view.model.get('scroll');
+        let gridBody = view.gridDiv.querySelector("div[ref=eBodyViewport]");
+        gridBody.scrollLeft = scroll[0]
+        gridBody.scrollTop = scroll[1]
+    });
+
+
+    const saveScroll = Helpers.debounce((left, top) => {
+        console.log(`bodyScroll(${left}, ${top})`);
+        view.model.set('scroll', [left, top]);
+        view.touch();
+    }, 300);
```

Comment:
add a comment next to the debounce of 300 - something like "if the user scrolls for 10 seconds, we will trigger the event only when he finished"

Reply:
done